webpack/components/SCCProductPage/components/SCCProductView/index.js

```diff
@@ -0,0 +1,234 @@
+      </Button>
+    </Tooltip>
+  );
+  return tree;
+};
+
+const setupTreeViewListItem = (tree, isRoot, editProductTree) => {
+  tree.key = 'view'.concat(tree.id.toString());
+  tree.customBadgeContent = [];
+  if (!tree.subscription_valid) addValidationStatusToTree(tree);
+  addReposToTree(tree);
+  addCheckBoxToTree(tree);
+  if (tree.product_id !== null) {
+    addKatelloLinkToTree(tree);
+    if (isRoot) {
+      addEditIcon(tree, editProductTree);
+    }
+  }
+  if ('children' in tree) {
+    tree.children = tree.children.map(setupTreeViewListItem, false);
+  }
+  return tree;
+};
+
+const filterDeep = (tree) => {
+  if (tree.product_id !== null) {
+    const filtered = clone(tree);
+    if ('children' in tree) {
+      filtered.children = filter(
+        tree.children,
+        (child) => child.product_id !== null
+      ).map(filterDeep);
+      if (filtered.children.length === 0) {
+        delete filtered.children;
+      }
+    }
+    return filtered;
+  }
+  return null;
+};
+
+const SCCProductView = ({
+  sccProducts,
+  editProductTreeGlobal,
+  subscriptionTaskId,
+}) => {
+  const editProductTree = (evt, productId) => {
+    editProductTreeGlobal(productId);
+  };
+  const sccProductsClone = cloneDeep(sccProducts);
+  // wrap actual iterator function into anonymous function to pass extra parameters
+  const [allProducts, setAllProducts] = useState(
+    sccProductsClone.map((tree) =>
+      setupTreeViewListItem(tree, true, editProductTree)
+    )
+  );
+  const [subscribedProducts, setSubscribedProducts] = useState(null);
+  const [showAll, setShowAll] = useState(true);
+  const [expandAll, setExpandAll] = useState();
+
+  const showSubscribed = () => {
+    if (subscribedProducts === null) {
+      const test = allProducts.map(filterDeep);
+      setSubscribedProducts(test);
+    }
+  };
+
+  const setShowAllFromChild = (showAllFromChild) => {
+    if (!showAllFromChild) {
+      showSubscribed();
+    }
+    setShowAll(showAllFromChild);
+  };
+
+  const setExpandAllFromChild = (expandAllFromChild) => {
+    setExpandAll(expandAllFromChild);
+  };
+
+  return (
+    <Card>
+      <CardTitle>{__('Subscribed SCC Products')}</CardTitle>
+      {sccProducts.length > 0 && (
+        <CardBody>
+          <Flex>
+            <SCCProductTreeExpander
+              setExpandAllInParent={setExpandAllFromChild}
+            />
+            <SCCSubscribedProductsExpander
+              setExpandAllInParent={setShowAllFromChild}
+            />
+          </Flex>
+          <Flex>
+            <TreeView
+              data={showAll ? allProducts : subscribedProducts}
+              allExpanded={expandAll}
+              hasChecks
+              hasBadges
+              hasGuides
+            />
+          </Flex>
+        </CardBody>
+      )}
+      {sccProducts.length === 0 && (
+        <CardBody>
+          {__(
+            'No SCC products imported. Please add new products by pressing the button below.'
```

Comment:
```suggestion
            'You currently have to SUSE products selected. Search and add SUSE products in the section below.'
```
Maybe even without `in the section`?